bdns/dns_test.go

```diff
@@ -473,131 +473,133 @@ func TestRetry(t *testing.T) {
 	servFailError := errors.New("DNS problem: server failure at resolver looking up TXT for example.com")
 	netError := errors.New("DNS problem: networking error looking up TXT for example.com")
 	type testCase struct {
+		description   string
 		maxTries      int
-		te            *testExchanger
+		testExchanger *testExchanger
 		expected      error
 		expectedCount int
 	}
-	tests := []*testCase{
-		// The success on first try case
+	testCases := []*testCase{
 		{
-			maxTries: 3,
-			te: &testExchanger{
+			"The success on first try case",
+			3,
+			&testExchanger{
 				errs: []error{nil},
 			},
-			expected:      nil,
-			expectedCount: 1,
+			nil,
```

Comment:
Can you restore the field names on these? It will make it easier to keep track of what the values correspond to.